pywaybiller/docs/ExternalAPIWaybillComment.md

```diff
@@ -0,0 +1,29 @@
+# ExternalAPIWaybillComment
+
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**comment_by_user_id** | **int** | The ID of the user that adds the comment to the waybill. Use GET Employments endpoint to query all available employments and get necessary &#x60;user_id&#x60; values. | 
+**comment** | **str** | Text of the comment. | 
+
+## Example
+
+```python
+from openapi_client.models.external_api_waybill_comment import ExternalAPIWaybillComment
+
+# TODO update the JSON string below
+json = "{}"
+# create an instance of ExternalAPIWaybillComment from a JSON string
+external_api_waybill_comment_instance = ExternalAPIWaybillComment.from_json(json)
+# print the JSON string representation of the object
+print ExternalAPIWaybillComment.to_json()
```

Comment:
should it be `print(ExternalAPIWaybillComment.to_json())` ? :thinking:

Reply:
It should... I am not sure if to try and fix the auto-generated docs, though. It also would need updating the strings above, as it says in todo, to provide a better example.